src-web/definitions/application.js

```diff
@@ -424,7 +428,13 @@ export function refreshApplicationComponents(appname, namespace, appNavConfigDat
           itemObj.name = <a href="#" onClick={() => resourceType == "application" ? displayApp(metadata.name, metadata.namespace) : displayDetail(appname, resourceType, metadata.name, metadata.namespace, applicationNamespace)}>
             {metadata.name}
           </a>;
-          itemObj.menuAction = getOverflowMenu(component, actionMap, applicationResourceData, appname, namespace);
+          if(resourceType.includes(msgs.get('resourceType.was'))){
```

Comment:
We should not need to check the type here.  We know the type is stored in the resourceType variable.  Can we do this?
itemObj.menuAction = getOverflowMenu(component, actionMap, getResourceData(resouceType), appname, namespace);